src/styles/components/_home.scss

```diff
@@ -18,7 +18,7 @@ $breakpoint-tablet: 820px;
         flex-basis: 50%;
 
         h1 {
-          font-size: 2.5rem;
+          font-size: map.get($font-sizes, "xl");
           font-family: Viga, sans-serif;
```

Comment:
Viga should be applied to h1 tags across the site instead of just on the home page. The h1 headers on the component pages (accordian, table, etc) are still in the oxygen font. I think it's safe to bump this css rule up in scope and add it to globals.scss, but I'd like a second opinion on that.